core/src/test/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/AttributeComponentSpec.scala

```diff
@@ -373,50 +375,111 @@ class AttributeComponentSpec extends TestDriverComponentWithFlatSpecAndMatchers
     }
   }
 
-  it should "extra data in entity attribute temp table should not mess things up" in withEmptyTestDatabase {
-    val workspaceId: UUID = UUID.randomUUID()
-    val workspace = Workspace(
-      "test_namespace",
-      "test_name",
-      workspaceId.toString,
-      "bucketname",
-      Some("workflow-collection"),
-      currentTime(),
-      currentTime(),
-      "me",
-      Map.empty,
-      false)
+  // TODO: what are good values to test here for parallelism?
+  List(4, 8, 16, 32, 64, 128, 256) foreach { parallelism =>
```

Comment:
This test is lifted from David's branch [here](https://github.com/broadinstitute/rawls/compare/mb-attrScratch...da-attrScratch?expand=1)

The changes I've made are:

1) use a larger thread pool based on the `parallelism` value
2) add an assertion to verify that we didn't run the saves with a single thread
3) change some of the parallelism values. 2 did not seem to work well, since sometimes it would run both saves on a single thread and fail the thread count assertion.

I am re-running these tests a bunch in GitHub to make sure that they aren't flaky.

Reply:
seeing that 4 fails (but rarely) 

going to remove that and start at 8